iroh/examples/collection/src/main.rs

```diff
@@ -0,0 +1,63 @@
+//! An example that serves an iroh collection from memory.
```

Comment:
I've added suggested changes to this file in a branch: https://github.com/n0-computer/iroh/tree/b5/iroh-examples

PTAL, and feel free to cherry-pick 331ea3a991c6eb35a25d60498424e1e717ab4e88